syncstorage-db-common/src/diesel.rs

```diff
@@ -4,10 +4,11 @@ use backtrace::Backtrace;
 use http::StatusCode;
```

Comment:
I'm not in love with keeping this `DbError` here in `syncstorage-db-common::diesel` -- it almost deserves to live in a `syncstorage-diesel` crate, but last thing I want is a new crate for a small module. We can always move this around later -- in the long term we should try to consolidate the spanner `DbError` with it anyways: (more about this [here](https://mozilla-hub.atlassian.net/browse/STOR-330?focusedCommentId=1159106))